internal/domain/indexer.go

```diff
@@ -353,6 +364,24 @@ func (p *IndexerIRCParseMatch) ParseTorrentName(vars map[string]string, rls *Rel
 	return nil
 }
 
+func (p *IndexerIRCParseMatch) ParseTorrentHash(vars map[string]string, rls *Release) error {
+	if p.TorrentHash != "" {
+		tmplTorrentHash, err := template.New("torrenthash").Funcs(sprig.TxtFuncMap()).Parse(p.TorrentHash)
+		if err != nil {
+			return err
+		}
+
+		var torrentHashBytes bytes.Buffer
+		if err := tmplTorrentHash.Execute(&torrentHashBytes, &vars); err != nil {
+			return errors.New("could not write torrent hash template output")
+		}
+
+		rls.TorrentHash = torrentHashBytes.String()
+	}
+
+	return nil
+}
```

Comment:
Isn't the hash in this case just a string? It doesn't look like it really needs to run with templates to execute

Reply:
Agreed. Saw too big there, updating.

Reply:
Because it's just a string and we don't have to do anything special with it, and it's part of the `vars`:

```yaml
vars:
  - postId
  - category
  - torrentHash
  - torrentName
```

it doesn't need any special handling as a parse function because these vars are available when we build the variables under `match`, like your new `magneturi` and `torrenturl` etc.

We do however have to explicitly set it like this [here in MapVars](https://github.com/autobrr/autobrr/blob/039162986261b64dac94fc372152432217509457/internal/domain/release.go#L967-L969) if we want to be able to use it later for anything.

```go
if torrentHash, err := getStringMapValue(varMap, "torrentHash"); err == nil {
	r.TorrentHash = torrentHash
}
```

Reply:
I see. Updated, I moved the torrentHash to the MapVars.